### ISSUE TITLE
add build support for backports.lzma

### ISSUE THREAD

**Issue by joachimmetz** [log2timeline/l2tdevtools#53]
- requires xz sources: http://tukaani.org/xz/
  - build: xz_win.sln
  - code C99 or later does not work with VS2008
- requires: https://pypi.python.org/pypi/backports.lzma/0.0.3
  - set up paths to build of xz sources
- tox need xz-headers to work

TODO:
- [x] ~~add dpkg support~~
- [x] ~~add osc support~~
- [ ] add MSI
- [ ] add PKG
- [ ] add support for pyliblzma and bz2 download file
- [ ] http://tukaani.org/xz/

**Comment by joachimmetz** [comment]
No longer needed Python 2 has been deprecated
